chemicaltoolbox/autodock_vina/prepare_ligand/prepare_ligand.xml

```diff
@@ -0,0 +1,43 @@
+<tool id="prepare_ligand" name="Prepare ligand" version="0.1.0">
+    <requirements>
+        <requirement type="package" version="1.5.6">mgltools</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+    <description> - Tool for prepare ligand with Autodock Vina</description>
+    <command interpreter="python"><![CDATA[
+        prepare_ligand.py -l "$ligand"
```

Comment:
the use of interpreter is deprecated and you should remove it.
Use `python $__tool_directory__/prepare_ligand.py` instead.